### PR TITLE
JBIDE-20347 - D&D install is not working, when link is dropped on Central

### PR DESCRIPTION
**Pull request by snjeza**
https://issues.jboss.org/browse/JBIDE-20347
D&D install is not working, when link is dropped on Central

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -415,8 +415,47 @@ function setIntegrationStackSupport(value) {
...
+function drop(event) {
+	event.preventDefault();
+	var data;
+	var msie = document.documentMode;
+	if (msie) {
+		data = event.dataTransfer.getData("Text");
+	} else {
+		data = event.dataTransfer.getData("text/uri-list");
+	}
+	var DOWNLOAD_JBOSS_ORG_JBOSSTOOLS_CENTRAL_INSTALL_CONNECTORS = "http://download.jboss.org/jbosstools/central/install?connectors=";
```
Comment (fbricon) [inline src/scripts/main.js:427]: L427-L437 : is this really necessary, given the same thing is implemented in eclipse?

Comment (snjeza) [inline src/scripts/main.js:427]: Yes. It fixes JBIDE-20348.

Comment (fbricon) [inline src/scripts/main.js:427]: Ok. Fair enough.

File: src/scripts/main.js [hunk 2]
```diff
@@ -415,8 +415,47 @@ function setIntegrationStackSupport(value) {
...
+	var DEVSTUDIO_REDHAT_COM_CENTRAL_INSTALL_CONNECTORS = "https://devstudio.redhat.com/central/install?connectors=";
+	var connectorIds;
+	if (data.indexOf(DOWNLOAD_JBOSS_ORG_JBOSSTOOLS_CENTRAL_INSTALL_CONNECTORS) === 0) {
+		connectorIds = data.replace(DOWNLOAD_JBOSS_ORG_JBOSSTOOLS_CENTRAL_INSTALL_CONNECTORS, "");
+	} else if (data.indexOf(DEVSTUDIO_REDHAT_COM_CENTRAL_INSTALL_CONNECTORS) === 0) {
+		connectorIds = url.replace(DEVSTUDIO_REDHAT_COM_CENTRAL_INSTALL_CONNECTORS, "");
+	} else if (data.indexOf(LEGACY_DEVSTUDIO_JBOSS_COM_CENTRAL_INSTALL_CONNECTORS) === 0) {
+		connectorIds = url.replace(LEGACY_DEVSTUDIO_JBOSS_COM_CENTRAL_INSTALL_CONNECTORS, "");
+	}
+	delegateToIDE("drop", connectorIds);
```
Comment (fbricon) [inline src/scripts/main.js:438]: might want to check connectorIds is not empty before delegating to the IDE

Comment (snjeza) [inline src/scripts/main.js:438]: IDE does it.

**Pull request merged**